azure-pipelines/pull-request-validation/pr-msal.yml

```diff
@@ -15,6 +15,7 @@ variables:
 - name: robolectricSdkVersion
   value: 33
 - group: MSIDLABVARS
+- group: devex-ciam-test
```

Comment:
What's the purpose of this group?

Reply:
https://identitydivision.visualstudio.com/Engineering/_library?itemType=VariableGroups
We can create the variable groups here and then use the variable in the pipeline directly to avoid public the mock api domain in the file.